### ISSUE TITLE
Change date modified

### ISSUE THREAD

**Issue by PickPacker** [holzschu/a-shell#209]
I’m attempting to change a date modified on a file using a-shell on iOS. I’m using the command 
touch -m “ccyymmddhh.ss” ~/Documents/filename.txt

Minus the quotes. it returns with no response and file not changed. It seems I’m not doing something right, or I can’t figure out the correct directory. Does anyone have an idea as to what path to use?

if I move the document and adjust the path accordingly just to iCloud Drive I get this:

touch: /private/var/mobile/Containers/Data/Application/1F9829C5-38B3-45B0-95B4-91AC30A
306F6/IMG_2433.JPG: Operation not permitted

I’ve tried copying that pathway and plugging in /Documents/filename.txt and it returns with this:

touch: /private/var/mobile/Containers/Data/Application/1F9829C5-38B3-45B0-95B4-91AC30/
Documents/IMG_2433.JPG: No such file or directory

What am I not getting?

**Comment by PickPacker** [comment]
Thanks in advanced. I haven’t coded anything since I was a teen, so it’s been a while.

**Comment by holzschu** [comment]
Hi, 
I've run some tests, and "touch -m 01010000 filename.txt" does change the modification date to 01/01, 00h00.
Trying to specify the year does change the year, but not in the way specified by the man page, so there's probably a bug there. Sometimes, specifying a full year with 4 digits results in the creation of a new file, named (in my case) `19990101000` (again, likely a bug). Running the same command on OSX produces the same result, though, so the bug is likely in the source code.

For your other question, file positions and permissions are different in iOS than the rest of the world. You cannot infer the position of the home directory from the position of the iCloud Drive, and you do not have read-write permission on the iCloud Drive by default (or any folder other than "On my iPad/a-Shell" and "iCloud folder/a-Shell"). You can get permission on any folder by selecting it using `pickFolder`, and then `touch` will work. 

Short version: if you see it with `ls`, you can change it with `touch`.

**Comment by PickPacker** [comment]
Yeah. The best outcome I got was a new file being created, which I had forgotten to mention. 

The file pathways you mentioned are they underscored with the spaces? 

Thank you for your time!

**Comment by PickPacker** [comment]
Got it! 
touch -mt YYMMDDhh.ss filename.txt
Apparently it seems you have to change the creation date as well as the the modified if you’re changing it to a date prior to it being created. Not positive, but that’s what I’m thinking.

**Comment by PickPacker** [comment]
I didn’t specify a pathway since I had the file in the a-shell folder.
